packages/web-react/src/components/Collapse/UncontrolledCollapse.tsx

```diff
@@ -5,6 +5,7 @@ import { useCollapse } from './useCollapse';
 import { useCollapseAriaProps } from './useCollapseAriaProps';
 
 const defaultProps = {
+  /** @deprecated ID will be made a required user input in the next major version. */
```

Comment:
If we have an issue with this Ii think it will be suitable to mention it as well.